lightning/src/chain/keysinterface.rs

```diff
@@ -840,6 +866,8 @@ impl KeysManager {
 					starting_time_nanos,
 				};
 				let secp_seed = res.get_secure_random_bytes();
+				let inbound_payment_key_bytes = res.get_secure_random_bytes();
```

Comment:
This will create a new value on each restart, which we don't want. Lets just hash a new child key?